src/app/project/documents/documents-tab.component.ts

```diff
@@ -172,11 +172,13 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
             // This code reorders the document type list defined by EAO (See Jira Ticket EAGLE-88)
             let copy_doctype = this.types;
             this.types = [];
-            // This order was created by mapping the doctype items from the database with the EAO defined ordered list
-            let docList_order = [0, 1, 2, 6, 10, 11, 14, 4, 3, 5, 13, 16, 15, 17, 18, 19, 7, 8, 9, 12];
             // We map the doctypes to put in the correct order as defined in doclist_order
-            docList_order.map((item, i) => {
-              this.types[item] = copy_doctype[i];
+            copy_doctype.map((item, i) => {
+              if (item.legislation === 2002) {
+                this.types[item.listOrder] = item;
```

Comment:
Should we try to use the listOrder directly? I would be worried that his could be fragile if the 2002 legislation values are ever change and are higher than 2018.

Reply:
I agree, it's fragile. Previously it was using a hardcoded list which broke it originally. I can try and change it up so it's checking legislation and doing a count of each to prevent magic numbers

Reply:
Actually, should just need:
this.types = _.sortBy(this.types, ['legislation', 'listOrder']);
so I can remove the whole sort